src/odemis/dataio/tiff.py

```diff
@@ -2782,6 +2782,7 @@ def _convert_thermo_fisher_to_odemis_metadata(metadata: str) -> dict:
         md[model.MD_ROTATION] = 0  # float(tfs_md[beam_type]["scanrotation"])
```

Comment:
Let's drop the MD_ROTATION entirely. The default is 0 anyway, and it highlights better that we don't really know. 
No need to keep the old version in a comment. We have git for that.